.ccache/ccache.conf

```diff
@@ -0,0 +1,26 @@
+# For complete documentation, see https://ccache.dev/manual/4.6.1.html#_configuration
+# Comments:
+# 1) By default, cache_dir is in $HOME/.ccache
```

Comment:
Should the cache_dir be inside the build dir?

Reply:
This is done intentionally.They must be inside this dir for few reasons,  (this is also the recommended default by ccache) with the main ones:

- To supshare cache between multiple projects/workspaces. Not everyone work with Git worktrees.
- Even when you clean or when you clone, ccache will go here. ccache is transperent to cleaning the build folder.